packages/net-stubbing/lib/server/util.ts

```diff
@@ -82,25 +83,28 @@ const caseInsensitiveGet = function (obj, lowercaseProperty) {
   }
 }
 
-export async function setBodyFromFixture (getFixtureFn: GetFixtureFn, staticResponse: BackendStaticResponse) {
+export async function setResponseFromFixture (getFixtureFn: GetFixtureFn, staticResponse: BackendStaticResponse) {
   const { fixture } = staticResponse
 
   if (!fixture) {
     return
   }
 
-  const data = await getFixtureFn(fixture.filePath, { encoding: fixture.encoding })
+  const data = await getFixtureFn(fixture.filePath, { encoding: fixture.encoding || null })
```

Comment:
so nice, the type is `| null` and code is `|| null`

Reply:
yep, to maintain compatibility with the cy.fixture code...